cyclient/scan_config/DefaultScanConfig.py

```diff
@@ -0,0 +1,5 @@
+from cyclient.scan_config.ScanConfigBase import ScanConfigBase
+
+
+class DefaultScanConfig(ScanConfigBase):
```

Comment:
move all the scan config to one class

Reply:
I did it since on dev mode we should run following route:
        self.SCAN_SERVICE_CONTROLLER_PATH = 'api/v1/scan'
        self.DETECTIONS_SERVICE_CONTROLLER_PATH = 'api/v1/detections'